thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -2000,7 +1992,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                 activity.inactiveStats.mergeStatistic(StNumDiskSeeks, diskSeeks);
             };
             COnScopeExit scoped(onScopeExitFunc);
-
+            CStatsScopedThresholdDeltaUpdater scoped2(activity.statsUpdater);
```

Comment:
trivial: 'scoped2' makes it look like there's another use case in the same scope, better to remove the '2' here.